.github/workflows/containerize-invehicle-stack-chariott.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: MIT
+name: Containerize In-Vehicle Stack - Chariott

```

Comment:
nit: Please add a blank line after the license banner.

Reply:
Good finding which slipped through on my end. In addition, the license of the rest of this blueprint is Apache 2.0, so it would be better to use the Apache 2.0 license for this workflow too. @ladatz can you change this?